src/tree/AppResourceTreeItem.ts

```diff
@@ -34,9 +33,11 @@ export class AppResourceTreeItem extends ResolvableTreeItemBase implements Group
         this.rootGroupConfig = <GroupNodeConfiguration><unknown>parent;
 
         this.data = resource;
-        this.commandId = 'azureResourceGroups.revealResource';
+        this.commandId = 'azureResourceGroups.revealResource'; // TODO: is this still needed?
```

Comment:
I think we don't need the command for RG, but we need to provide a way for other extensions to use "revealResource".   A user may want to reveal a resource on the tree especially after creation.

I think we could let them use `revealResource` like so:
https://github.com/microsoft/vscode-azurestaticwebapps/commit/4ec265304fb94485d5389c820c5ba25e7e82265e#diff-a448840ab47b15ffc93675e7af136d99e1bc309a7ed03d64f126ea049aa54803R15

Reply:
Oh, my impression was that it revealed and highlighted it in the treeView even if the node was collapsed.

Reply:
Ok, I'll need to review, this might be the wrong command ID. Ultimately the one we need is whichever is the one opening the raw JSON.

Reply:
Ok, the correct command is `azureResourceGroups.viewProperties`, I have updated it.